code/modules/mob/living/carbon/shock.dm

```diff
@@ -1,47 +1,150 @@
+#define SOFTCRIT_TRAUMATIC_SHOCK	50
+#define HARDCRIT_TRAUMATIC_SHOCK	100
+#define SHOCK_STAGE_BUFFER			50
```

Comment:
It's not aligned ;-;

Reply:
It. Is. A. Travesty.
![travesty](https://user-images.githubusercontent.com/61743710/116100282-0ea24280-a6ad-11eb-81e7-fd8b4bd7c0b3.png)

Reply:
It's because you used tabs, github tabs are not the same size than in Visual Studio ;-;